src/menu/components/context_menu.c

```diff
@@ -41,6 +42,7 @@ bool component_context_menu_process (menu_t *menu, component_context_menu_t *cm)
         } else {
             cm->hide_pending = true;
         }
+        wav64_play(&sfx_exit, SFX_CHANNEL);
```

Comment:
I would rather make a function in `sound.c` to abstract the libdragon API and to avoid passing `&sfx_...` around via `extern`. Instead, we could just have an enum with list of possible sounds to play. It would look like this: `sound_play_effect(SFX_EXIT);`

Reply:
e.g. an enum like:
```typedef enum {
    SFX_CURSOR,
    SFX_ERROR,
    SFX_ENTER,
    SFX_EXIT,
    SFX_SETTING
} sound_effect_t```